### PR TITLE
Hide mce tooltips

### PR DESCRIPTION
**Pull request by adamsilverstein**
also, slight refactoring of mce startup

### CODE REVIEW

**Review by paulschreiber — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts.

**Comment by adamsilverstein** [comment]
@paulschreiber resolved merge conflicts

File: src/react/components/TinyMCEEditor.js [hunk 1]
```diff
@@ -37,22 +38,29 @@ class TinyMCEEditor extends Component {
...
+      this.editorSettings.tinymce.setup = (editor) => {
         const stateContent = this.props.editorContainer.getContent();
-        tinymce.activeEditor.clearAuthors = this.props.clearAuthors;
-        tinymce.activeEditor.clearHeadline = this.props.clearHeadline;
-        if (stateContent && stateContent !== '' && stateContent !== '<p></p>') {
-          tinymce.activeEditor.setContent(stateContent);
-        }
-      }, 500);
+        editor.clearAuthors = this.props.clearAuthors;
+        editor..clearHeadline = this.props.clearHeadline;
```
Comment (paulschreiber) [inline src/react/components/TinyMCEEditor.js]: should this have one `.`?

Comment (adamsilverstein) [reply src/react/components/TinyMCEEditor.js]: oops, no - will correct.

**Comment by adamsilverstein** [comment]
@paulschreiber - corrected, the changes in that file were unrelated

**Review by paulschreiber — APPROVED** [review]

File: assets/dashboard/app.css [hunk 2]
```diff
@@ -110,3 +110,7 @@ body.wp-admin .liveblog-entry {
   border: 1px solid #ccc;
   margin-top: 5px;
 }
+
+body.wp-admin.liveblog.post-type-fte_liveblog.wp-core-ui .mce-tooltip {
```
Comment (paulschreiber) [inline assets/dashboard/app.css]: This hard-codes a reference to `fte_liveblog`, which won't work for others. Please use a different selector.

**Review by paulschreiber — CHANGES_REQUESTED** [review]

File: assets/dashboard/app.css (hunk 2 above)
Comment (adamsilverstein) [reply assets/dashboard/app.css]: Ok, to make this more generic i can rely on the body class that the plugin adds for live posts.

Question: what do you mean by "others" - other post types? are you using the liveblogging feature elsewhere or planning to?

Comment (paulschreiber) [reply assets/dashboard/app.css]: Others: people not FiveThirtyEight who will also use this liveblog. (We want to merge these changes in to master, and not have a fork.)

Comment (adamsilverstein) [reply assets/dashboard/app.css]: 👍 I'll fix

Comment (adamsilverstein) [reply assets/dashboard/app.css]: Fixed in https://github.com/paulschreiber/liveblog/pull/19/commits/7bbcc471b4d379ef315af7fc83a4a02480b7e3f1 where I removed all reverences to post-type-fte_liveblog in the css. the existing `.liveblog` targeting is specific enough.

**Pull request merged**